README.md

```diff
@@ -1,2 +1,14 @@
 # toolhunt-ui
-Frontend Repository for Toolhunt. Toolhunt is a web application for editing Toolhub records in a fun and easy way. It allows users to view and add missing fields for tools in Toolhub. This is an Outreachy Internship project.
```

Comment:
It would be useful to link to the backend repo here.

Reply:
Great let me add it right now